de-DE.json

```diff
@@ -1,7 +1,7 @@
 {
   "1 {0} = {1} {2}": "1 {0} = {1} {2}",
   "Details": "Details",
-  "Note that if you use Typhon wallet in HD mode, you will only see orders sent from current address. For\n                    best experience, please use Typhon in single-address mode.": "Note that if you use Typhon wallet in HD mode, you will only see orders sent from current address. For\n                    best experience, please use Typhon in single-address mode.",
+  "Note that if you use Typhon wallet in HD mode, you will only see orders sent from current address. For\n                    best experience, please use Typhon in single-address mode.": "Beachten Sie, dass Sie bei der Verwendung von Typhon Wallet im HD-Modus nur die von der aktuellen Adresse gesendeten Aufträge sehen.\nUm die beste Nutzererfahrung zu erzielen, verwenden Sie bitte Typhon im Einzeladressmodus.",
```

Comment:
I'm unsure about the newline and spaces. It's seems to need adjustments by the dev team, doesn't it?

Reply:
yes it should be adjusted in the UI not here
besides if we don't have `white-space` CSS the `\n` wouldn't work anyways

Reply:
I removed the newline `\n` in 4c489680510f0d5887f40d305b5f4e49aedb15d0